Test/Framework/Visual/VertexBufferTest.cs

```diff
@@ -121,7 +121,36 @@ public void GetPosition(bool dynamic)
                 Assert.AreEqual(savedData[2].Position, readData[2]);
                 Assert.AreEqual(savedData[3].Position, readData[3]);
             };
-            Game.RunOneFrame();
+            Game.Run();
+        }
+
+        //[TestCase(true)]
```

Comment:
Presumably we do eventually want `GetData` for dynamic vertex buffers (and `SetData` with a `vertexStride`), for XNA compatibility, whether or not it's a good idea to actually use it? But I was hoping to leave that for the future - I haven't ever used `DynamicVertexBuffer` myself.